blockchain/Cargo.toml

```diff
@@ -29,6 +29,12 @@ path = "../zkvm"
 [dependencies.starsig]
 path = "../starsig"
 
+[dependencies.p2p]
+path = "../p2p"
```

Comment:
if we only need this for the test, it should be in dev-dependencies. But i think we can just use Vec<u8> and not pull p2p/tokio/bytes.

Reply:
p2p was removed from dependencies in blockchain